provider/scalewayProvider.js

```diff
@@ -57,15 +58,26 @@ class ScalewayProvider {
     }
   }
 
+  setApiURL(options) {
+    if (options['scw-region']) {
+      this.scwRegion = options['scw-region'];
+    } else if (process.env.SCW_REGION) {
+      this.scwRegion = process.env.SCW_REGION;
+    } else {
+      this.scwRegion = this.serverless.service.provider.scwRegion || 'fr-par';
```

Comment:
maybe `fr-par` should be added to constants as DEFAULT_REGION or something like that ?